apps/frontend-manage/.env.prd

```
@@ -0,0 +1,24 @@
+COOKIE_DOMAIN=.klicker.prd.df-app.ch
+API_DOMAIN=backend-sls$COOKIE_DOMAIN
+
+NEXT_PUBLIC_AVATAR_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/avatars"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)